otx/algorithms/classification/tasks/train.py

```diff
@@ -157,6 +157,11 @@ def _init_train_data_cfg(self, dataset: DatasetEntity):
                 ),
             )
         )
+        if len(dataset.get_subset(Subset.UNLABELED)):
```

Comment:
how about assign dataset.get_subset(Subset.UNLABELED) to variable and reuse it?

Reply:
I wonder if it's better in code implementation. I'm curious. I asked because I don't know.

Reply:
ah, I just think that if using variable then, we don't call dataset.get_subset(Subset.UNLABELED) twice.

Reply:
Yeah, makes sense. I made a function to get unlabeled dataset. Please check and resolve related conversations if no objections.